.github/workflows/integration-tests.yml

```diff
@@ -7,16 +7,11 @@ on:
     paths-ignore:
       - '**.md'
   pull_request:
-  # Allow manually triggering the workflow.
-  workflow_dispatch:
```

Comment:
Why was this removed?

Reply:
I'm not a big fan of manually triggering workflows, because they might have unintended consequences. I find it way better and easier to track to tie executions to a push/pull request.